code/modules/unit_tests/servingtray.dm

```diff
@@ -0,0 +1,38 @@
+/**
+ * Check that standard food items fit on the serving tray
+ */
+/datum/unit_test/servingtray/Run()
+	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
+	var/obj/structure/table/the_table = allocate(/obj/structure/table)
+	var/obj/item/storage/bag/tray/test_tray = allocate(/obj/item/storage/bag/tray)
+	var/obj/item/reagent_containers/food/banana = allocate(/obj/item/reagent_containers/food/snacks/grown/banana)
+	var/obj/item/food/the_bread = allocate(/obj/item/food/breadslice)
+	var/obj/item/reagent_containers/food/sugarcookie = allocate(/obj/item/reagent_containers/food/snacks/sugarcookie)
+
+	TEST_ASSERT_EQUAL((the_bread in test_tray.contents), FALSE, "The bread is on the serving tray at test start")
+
+	// set the tray to single item mode the dirty way
+	var/datum/component/storage/tray_storage = test_tray.GetComponent(/datum/component/storage)
+	tray_storage.collection_mode = 0 //COLLECT_ONE
```

Comment:
Can you move out `COLLECT_ONE` and the other defines to something like `code/__DEFINES/storage.dm`? I'm uncomfortable relying on the value here.

Reply:
Fair, i will add this tot he next update where it passes it's own test